packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_table_row.dart

```diff
@@ -49,9 +51,10 @@ class LoggingTableRow extends StatefulWidget {
     // TODO(danchevalier): Improve row2 height by manually flowing metadas into another row
     // if theyoverflow.
     final row2Height = calculateTextSpanHeight(
-      TextSpan(text: text, style: metadataStyle),
+      TextSpan(text: 'short text for now', style: metadataStyle),
```

Comment:
remove this hard coded string

Reply:
Metadata will be handled in a future PR. I'll just leave this for now.